requirements_prophet.txt

```diff
@@ -1,2 +1,2 @@
 pystan==2.19.1.1
-fbprophet
\ No newline at end of file
+prophet
```

Comment:
This looks promising! @abitrolly can you force the version to be `==1.1`?

Reply:
Done. This should speed up Docker builds by some 20+ minutes. However, I am not sure if `prophet` is still in MindsDB defaults.